serverless/sql-databases/reference-content/serverless-sql-databases-overview.mdx

```diff
@@ -42,3 +42,34 @@ After receiving no query for 5 minutes, a Serverless SQL Database will enter int
 
 While your database is in idle state, you will not be billed for compute resources (vCPU and RAM). You will however still be billed for storage resources.
 
+## Compatibility
+
+Serverless SQL Database is compatible with most PostgreSQL-compatible clients and tools. Due to connection pooling support, some minor features are disabled or will not work as expected. Check out the [Known differences](/serverless/sql-databases/reference-content/known-differences/)
+
+If you require strict compatibility with all PostgreSQL features, you can use [Managed Databases for PostgreSQL](/managed-databases/postgresql-and-mysql/quickstart/) instead.
+
+## Configuration requiring manual actions
```

Comment:
As an afterthoughts, in terms of order, I would swap "Authentication and access management" and "Configuration requiring manual actions" as it seems more important.